packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -270,12 +288,15 @@ const Breakouts = WebexPlugin.extend({
             breakouts[sessionId][BREAKOUTS.SESSION_STATES.ASSIGNED] = false;
             breakouts[sessionId][BREAKOUTS.SESSION_STATES.ALLOWED] = false;
             breakouts[sessionId][BREAKOUTS.SESSION_STATES.ASSIGNED_CURRENT] = false;
-            breakouts[sessionId][BREAKOUTS.SESSION_STATES.REQUESTED] = false;
           }
 
           breakouts[sessionId][state] = true;
         });
       });
+
+      if (some(requestedBreakoutSessions, ['sessionType', BREAKOUTS.SESSION_TYPES.MAIN])) {
```

Comment:
question: What was the reason for needing to change the way the REQUESTED type was processed? Why wasn't something like this possible?
```ts
if (this.getMainSession().requested) {
  this.trigger(BREAKOUTS.EVENTS.ASK_RETURN_TO_MAIN);
}
```

Reply:
I was thinking that I could determine whether there was a `REQUESTED` directly from DTO, and `breakout.requested` was a bit redundant, so I decided to erase it. Thus reducing unwanted side effects. 
**But it is not necessary to change it, I have rolled back the extra changes.**

Reply:
I need to check `allowBackToMain` before calling `getMainSession`, otherwise getMainSession will throw an exception. Because if `allowBackToMain` is false, then there is no `main session`.